.github/workflows/rust.yml

```diff
@@ -248,3 +249,38 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: EmbarkStudios/cargo-deny-action@v1
+
+  move-prover:
+    name: move-prover
+    needs: diff
+    if: needs.diff.outputs.isMove == 'true'
+    runs-on: [ubuntu-ghcloud]
+
+    steps:
+      - name: Sleep for 20 minutes
+        run: sleep 1200
```

Comment:
Does it mean that this check will now always run for 20 minutes? I know very little about this setup but I did see the `timeout-minutes: 45` clause earlier in this file. Is there a difference between the two options?

Reply:
Oh yeah, setting time out is more consistent with the rest of the steps. Let me update.